pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1454,30 +1449,19 @@ static __always_inline int init_event_data(event_data_t *data, void *ctx)
 // INTERNAL: FILTERING -----------------------------------------------------------------------------
 
 static __always_inline int
-uint_filter_matches(bool filter_out, void *filter_map, u64 key, u32 less_idx, u32 greater_idx)
+uint_filter_matches(bool filter_out, void *filter_map, u64 key, u64 lower_than, u64 greater_than)
 {
     u8 *equality = bpf_map_lookup_elem(filter_map, &key);
     if (equality != NULL)
         return *equality;
 
-    if (!filter_out)
-        return 0;
-
-    u64 *lessThan = bpf_map_lookup_elem(&inequality_filter, &less_idx);
-    if (lessThan == NULL)
-        return 1;
-
-    if ((*lessThan != LESS_NOT_SET) && (key >= *lessThan))
+    if ((lower_than != LOWER_NOT_SET) && (key >= lower_than))
```

Comment:
This would break backwards compatibility, but just want to note, we technically set this behavior for the `<` operation, not `<=`. Maybe it's worth changing to be in line (note that in #1750 I allow for both operations).